compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -982,21 +982,21 @@ class FormalActualMap {
 
 template<> struct stringify<chpl::resolution::TypedFnSignature::WhereClauseResult>
 {
-  void operator()(std::ostream &stringOut,
+  void operator()(std::ostream& streamOut,
                   chpl::StringifyKind stringKind,
                   const chpl::resolution::TypedFnSignature::WhereClauseResult& stringMe) const {
     switch(stringMe) {
       case 0:
```

Comment:
I think these should be using the enum and not an int. I also think this is in the chpl namespace so you can drop the `chpl::`. And maybe do a scoped (inside the function body) `using WhereClauseResult = resolution::TypedFnSignature::WhereClauseResult;` so that the `case` branches aren't enormous